vmdb/spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -46,7 +48,7 @@ def assert_specific_container
       # :ems_ref       => "a7566742-e73f-11e4-b613-001a4a5f4a02_heapster_kubernetes/heapster:v0.9",
       :name          => "heapster",
       :restart_count => 0,
-      :image         => "kubernetes/heapster:v0.9",
+      :image         => "kubernetes/heapster:v0.11.0",
```

Comment:
APIs are changing all over the place here :smile: so basically atm you have to find an hepaster version that is capable of running on your kubernetes cluster. I had to upgrade heapster for that reason. In some way we'll stabilize on what heapster version to use in the future (except for erratas).